libc/socket/inet-socket.c

```diff
@@ -0,0 +1,172 @@
+/*
+ * Phoenix-RTOS
+ *
+ * test-libc-socket
+ *
+ * inet socket tests
+ *
+ * Copyright 2021, 2024 Phoenix Systems
+ * Author: Ziemowit Leszczynski, Adam Debek
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/uio.h>
+#include <sys/wait.h>
+
+#include "common.h"
+#include "unity_fixture.h"
+
+char data[DATA_SIZE];
+char buf[DATA_SIZE];
```

Comment:
Is this used anywhere?
```suggestion
char buf[DATA_SIZE];
```

Reply:
this is defined as extern in common.h so should probably be placed in common.c, please remove it from here.

Still, the `data` is being used only once in `common.c` so it probably doensn't need to be global (extern), just local (static) for the test.